src/metax_api/models/directory.py

```diff
@@ -151,42 +152,21 @@ def calculate_byte_size_and_file_count_for_cr(self, cr_id, directory_data):
         /root/experiments/phase_2
         These do have the numbers calculated. These are the directories that this method is executed
         on initially.
```

Comment:
There is a NOTE in this comment block that states that leading prefix directories are not handled. This should be removed also.